packages/omgx/wallet-frontend/src/components/transaction/Transaction.js

```diff
@@ -99,7 +100,10 @@ function Transaction ({
           styles.dropDownContainer : dropDownBoxInit ? styles.dropDownInit : styles.closeDropDown}
       >
         <div className={styles.title}>
-          <div>{detail.l1Hash}</div></div>
+          <a className={styles.href} href={`https://rinkeby.etherscan.io/tx/${detail.detailedL1Hash}`} target="_blank" rel="noopener noreferrer">
+            {detail.l1Hash}
+          </a>
+        </div>
         <div className={styles.content}>
```

Comment:
Instead of using hard coded link can you please use the masterConfig and method called ChainLink?